docs/api.rst

```diff
@@ -255,6 +257,21 @@ Create a :ref:`Stream` containing a single event at a specific time. ::
 
   at(3, x): --x|
 
+.. _periodic:
+
+periodic
+````````
+
+.. code-block:: haskell
+
+  periodic :: Period -> Stream void
+
+Create an infinite :ref:`Stream` containing events that occur at a specified period. ::
+
+  periodic(3): .--.--.--.-->
```

Comment:
Is the `.` here confusing?  Maybe we should just use `x`?

Reply:
yes and yes